### PR TITLE
Add ICEcoder

### PR DESCRIPTION
**Pull request by Stylix58**

### CODE REVIEW

**Comment by Stylix58** [comment]
By the power of systemd, I invoke The Great App CI to test this Pull Request!

**Comment by tituspijean** [comment]
... nope :) Take a breath and read the instructions I sent you in the App dev room. :)

**Review by Stylix58 — COMMENTED** [review]
I am ok with this!

File: apps.json [hunk 1]
```diff
@@ -1424,6 +1424,16 @@
...
     },
+    "icecoder": {
+        "branch": "master",
+        "category": "dev",
+        "revision": "HEAD",
+        "state": "inprogress",
+        "subtags": [
+            "programming"
+        ],
+        "url": "https://github.com/YunoHost-Apps/icecoder_ynh",
```
Comment (tituspijean) [inline apps.json:1435]: There is an extra comma right before the closing bracket.

**Review by tituspijean — CHANGES_REQUESTED** [review]

File: apps.json (hunk 1 above)
Comment (Stylix58) [reply apps.json]: Ok!

**Review by ericgaspar — APPROVED** [review]

**Review by tituspijean — APPROVED** [review]

**Pull request merged**